schemas/schema-schema.ipldsch

```diff
@@ -116,17 +116,17 @@ type Type union {
 ## TODO: not actually sure we'll need to declare this.  Only usage is
 ## in the Type union representation details?
 type TypeKind enum {
-	| "bool"
-	| "string"
-	| "bytes"
-	| "int"
-	| "float"
-	| "map"
-	| "list"
-	| "link"
-	| "union"
-	| "struct"
-	| "enum"
+	| bool
```

Comment:
My first inclination would be to make these Snake/TitleCase, on the logic that so far lowercase is showing up in keywords and field names.  Thoughts?

(Docs above also have SCREAMING case, which I'm just never a fan of, personally.)

Reply:
capitalised first character